packages/cli/src/databases/entities/Project.ts

```diff
@@ -11,10 +12,30 @@ export class Project extends WithTimestampsAndStringId {
 	@Column({ length: 255, nullable: true })
 	name?: string;
 
+	get actualName(): string {
+		if (this.name) {
+			return this.name;
+		}
+
+		const owningRelation = this.projectRelations?.find(
+			(pr) => pr.role === 'project:personalOwner' && pr.user !== undefined,
+		);
+		if (owningRelation) {
+			const user = owningRelation.user;
+			return `${user.firstName} ${user.lastName} <${user.email}>`; // generate name
+		}
+
+		console.trace(JSON.stringify(this.projectRelations, null, 2));
+
+		throw new ApplicationError(
+			'You asked for the actual name of a personal project. I can only generate the actual name if `projectRelations: { user: true }` is part of the relations when querying the database. Please make sure to add this to your query.',
+		);
+	}
+
 	@Column({ length: 36 })
 	type: ProjectType;
 
-	@OneToMany('ProjectRelation', 'project')
+	@OneToMany('ProjectRelation', 'project', { eager: true })
```

Comment:
This made getting all tests to pass relatively easy. If we disable this there are more routes that need to be fixed manually by adding the necessary relations to the repository methods.